src/main/java/seedu/address/model/person/github/User.java

```diff
@@ -21,6 +23,7 @@ public class User {
             + "    - have each domain label consist of alphanumeric characters, separated only by hyphens, if any.";
 
     private String username = "";
+    private RepoList repoList = new RepoList();
```

Comment:
I think you miss out `repoList` in constructor.

Reply:
repoList is populated using GitHub API, so Arnav and I were thinking of initializing the repoList separately.

Reply:
Yes I understood, but now there is no way to set `repoList` for `User` class.